custom-weather-card-chart.js

```diff
@@ -78,6 +78,9 @@ class WeatherCardChart extends Polymer.Element {
   static get template() {
     return Polymer.html`
       <style>
+        ha-card {
+          padding: 16px 0 0 0;
```

Comment:
I think you should remove the `.main`'s `margin-top: -10px;` instead of adding a new padding

Reply:
Ohh I see your point

Reply:
anyway, this PR has been open for more than 2 years without any consideration, so...